### PR TITLE
wip: gallery functionality

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**
I will appreciate a wave of constructive criticism :)

### CODE REVIEW

**Comment by FrantisekMichalSebestyen** [comment]
How to test:
`src/renderer/images`: here goes `test.png` thumbnail of artwork
`src/renderer/public/details` here goes: 

- `images` images of ui icons + sngskin folder
- `test/tiles`
- `test/test.xml`
- `test/sng.css`

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,6 @@ dist
 build
 
 .vscode
-.idea
\ No newline at end of file
+.idea
+src/renderer/public/details
+src/renderer/images
```
Comment (igor-kamil) [inline .gitignore:8]: also `src/renderer/images` should be in public -> `src/renderer/public/images`

File: src/renderer/views/ArtworkDetail.vue [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+<script lang="ts" setup>
+import { onMounted } from "vue";
+import { loadScript } from "vue-plugin-load-script";
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+
+onMounted(() => {
+  const id = route.params.id
+  loadScript("/object2vr_player.js")
```
[the remark this answers is not in the document]
Comment (igor-kamil) [reply src/renderer/views/ArtworkDetail.vue:14]: just don't forget about it

File: src/renderer/views/Catalogue.vue [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+    </div>
+    <div
+      class="w-full h-[72px] flex items-center justify-center p-6"
+    >
+      <svg class="inline-block h-6 w-6 fill-none" viewBox="0 0 24 24">
+        <path
+          d="M12.8266 12.8266H22.3649V16.7691H14.4618V18.4042H22.3649V22.3467H12.8266V12.8266ZM12.8266 2.79788L21.2021 11.1734H12.8266V2.79788ZM1.63512 1.63512H11.1734V11.1734H1.63512V1.63512ZM13.9894 1.63512H22.3649V10.0106L13.9894 1.63512ZM0 0V12.8266H11.1734V22.3649H1.63512V14.4618H0V24H24V0H0Z"
+          fill="#050505"
+        />
+      </svg>
```
Comment (igor-kamil) [inline src/renderer/views/Catalogue.vue:24]: at least SNG svg logo could be reused component 😉 
(not blocker)

File: .gitignore (hunk 1 above)
Comment (igor-kamil) [reply .gitignore:8]: maybe even better to put `.gitignore` inside the folders... (same like for `fonts`)

File: src/renderer/views/Catalogue.vue (hunk 3 above)
Comment (FrantisekMichalSebestyen) [reply src/renderer/views/Catalogue.vue:24]: will change in NSNG-344-styles

File: src/renderer/views/ArtworkDetail.vue [hunk 4]
```diff
@@ -22,4 +22,9 @@ onMounted(() => {
       obj.readConfigUrl(`/details/${id}/${id}.xml`);
     });
 });
+
+onBeforeUnmount(() => {
+  unloadScript("/object2vr_player.js");
+  unloadScript("/skin.js");
+});
```
Comment (igor-kamil) [inline src/renderer/views/ArtworkDetail.vue:29]: 👍

**Review by igor-kamil — APPROVED** [review]
👌

**Pull request merged**